qml/items/PlayField.qml

```diff
@@ -243,10 +223,13 @@ Item {
 
     function resetField(forceReset)
     {
-        if (!forceReset && !solved())
+        if (!solved() && !forceReset)
```

Comment:
This is actually a regression -- assuming the engine does lazy evaluation (and no other optimizations) the (-) code performs a bit better, checking the variable first.